docs/resources/Audit_Log.md

```diff
@@ -127,7 +127,7 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 |-------------------------------|-----------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------|
 | name                          | any                                                                                                                         | string                                                                               | name changed                                                                                                                                            |
 | description                   | [guild](#DOCS_RESOURCES_GUILD/guild-object) or [sticker](#DOCS_RESOURCES_STICKER/sticker-object)                            | string                                                                               | description changed                                                                                                                                     |
-| icon_hash                     | [guild](#DOCS_RESOURCES_GUILD/guild-object)                                                                                 | string                                                                               | icon changed                                                                                                                                            |
+| icon_hash                     | [guild](#DOCS_RESOURCES_GUILD/guild-object) or [role](#DOCS_TOPICS_PERMISSIONS/role-object)                                 | string                                                                               | icon changed                                                                                                                                            |
```

Comment:
Another field in the audit log is `unicode_emoji` which allows for the name of a unicode emoji to be the role icon.

Reply:
Done

Reply:
While I'm on it.
Shall I sort it via abc

Reply:
tbh, to do more testing on this, I'm developing an own lib. Is it possible to get a guild override for role icons ^^
It's badly not yet in the normal rollout, so I'm kinda blind and have to relay on other users.
@jkcailteux

Reply:
^